samples/iterable_dataset/multiple_workers.py

```diff
@@ -0,0 +1,41 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""Loads data samples using multiple workers"""
+
+from azstoragetorch.datasets import IterableBlobDataset
+from torch.utils.data import DataLoader
+
+
+def load_with_workers():
+    # Update URL with your own Azure Storage account and container name
+    CONTAINER_URL = (
+        "https://<my-storage-account-name>.blob.core.windows.net/<my-container-name>"
+    )
```

Comment:
For the `CONTAINER_URL` let's define it outside of the function as part of the top-level module. In general, when a variable is capital cased in Python, it serves as a constant and generally should be defined at the module level.